scripts/push_package.sh

```diff
@@ -0,0 +1,18 @@
+# The script will stop if any intermediate file raises an error.
+set -e
+
+# TODO: Dynamically generate names (The order of installing and pushing is important here.)
+modulesList=("src/openqaoa-core" "src/openqaoa-qiskit" "src/openqaoa-braket" "src/openqaoa-pyquil" "src/openqaoa-azure")
+
+python scripts/test_version.py
+
+for entry in "${modulesList[@]}"; do
+    echo "processing $entry/setup.py"
+    cd $entry
+    pip install .
+    # python -m build
+    python3 setup.py -q sdist
+    python3 setup.py -q bdist_wheel
+    twine upload --repository testpypi dist/* --username $1 --password $2
```

Comment:
The idea is for this script to be run through an action, right? And then $1 and $2 would be input called when the script is executed, right?

Reply:
Yes! $1 and $2 would be inputs to the script, most likely the secret keys, in order to upload to testpypi and then maybe pypi in the future.